include/NaluParsedTypes.h

```diff
@@ -54,6 +54,20 @@ struct SpecDissRate {
   {}
 };
 
+struct GammaInf {
+  double gamma_;
+  GammaInf()
+    : gamma_(1.0)
```

Comment:
Most of these default to `0.0`. Are you sure you want both of these Gamma ones to default to 1.0?

Reply:
Thanks for your comment. I was using GammaInf to set the value of gamma to unity at the inflow (see GammaEquationSystem.C). I believe that Shreyas suggested that we set the inflow BC using a 'GammaInf struct'.